ios/RCTPSPDFKit/Converters/RCTConvert+PSPDFAnnotation.m

```diff
@@ -23,8 +23,8 @@ @implementation RCTConvert (PSPDFAnnotation)
         [annotationsJSON addObject:annotationDictionary];
       }
     } else {
-      // We only generate Instant JSON data for attached annotations. When an annotation is deleted, we only set the annotation uuid.
-      [annotationsJSON addObject:uuidDict];
+      // We only generate Instant JSON data for attached annotations. When an annotation is deleted, we only set the annotation uuid and name.
+      [annotationsJSON addObject:@{@"uuid" : annotation.uuid, @"name" : annotation.name ?: annotation.uuid}];
```

Comment:
We shouldn't fall back on the UUID for the name unless this also works similarly in other places of the wrapper. I think having this explicitly be `null` would be cleaner.

Reply:
Good point! Addressed!